diff --git a/MODULE.bazel b/MODULE.bazel
index 2649521..7a652c1 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -62,7 +62,7 @@ python.toolchain(
     python_version = PYTHON_VERSION,
 )
 
-pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip", dev_dependency = True)
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
 pip.parse(
     hub_name = "pip_score_persistency_venv_test",
     python_version = PYTHON_VERSION,
