diff --git a/MODULE.bazel b/MODULE.bazel
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -53,10 +53,10 @@ python.toolchain(
     python_version = PYTHON_VERSION,
 )
 
-pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip", dev_dependency = True)
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
 pip.parse(
-    hub_name = "pip_score_venv_test",
+    hub_name = "pip_score_persistency_venv_test",
     python_version = PYTHON_VERSION,
     requirements_lock = "//score/kvs/tests/test_cases:requirements.txt.lock",
 )
-use_repo(pip, "pip_score_venv_test")
+use_repo(pip, "pip_score_persistency_venv_test")
diff --git a/score/kvs/tests/test_cases/BUILD b/score/kvs/tests/test_cases/BUILD
--- a/score/kvs/tests/test_cases/BUILD
+++ b/score/kvs/tests/test_cases/BUILD
@@ -9,7 +9,7 @@
 # https://www.apache.org/licenses/LICENSE-2.0
 #
 # SPDX-License-Identifier: Apache-2.0
 # *******************************************************************************
-load("@pip_score_venv_test//:requirements.bzl", "all_requirements")
+load("@pip_score_persistency_venv_test//:requirements.bzl", "all_requirements")
 load("@rules_python//python:pip.bzl", "compile_pip_requirements")
 load("@score_tooling//python_basics:defs.bzl", "score_py_pytest", "score_virtualenv")
