diff --git a/MODULE.bazel b/MODULE.bazel
index 7a652c1..0fe4d1e 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -41,7 +41,7 @@ bazel_dep(name = "google_benchmark", version = "1.9.5", dev_dependency = True)
 bazel_dep(name = "googletest", version = "1.17.0.bcr.2", dev_dependency = True)
 bazel_dep(name = "score_bazel_cpp_toolchains", version = "1.0.3", dev_dependency = True)
 bazel_dep(name = "score_sbom", version = "0.1.1", dev_dependency = True)
-bazel_dep(name = "score_rules_imagefs", version = "0.1.0", dev_dependency = True)
+bazel_dep(name = "score_rules_imagefs", version = "0.1.0")
 bazel_dep(name = "score_rust_policies", version = "0.0.5", dev_dependency = True)
 bazel_dep(name = "score_toolchains_rust", version = "0.10.0", dev_dependency = True)
 bazel_dep(name = "score_tools", version = "0.0.3", dev_dependency = True)
