diff --git a/MODULE.bazel b/MODULE.bazel
index 1aae83e..9b65df4 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -44,7 +44,7 @@ bazel_dep(name = "score_sbom", version = "0.1.1", dev_dependency = True)
 bazel_dep(name = "score_rules_imagefs", version = "0.1.0")
 bazel_dep(name = "score_rust_policies", version = "0.0.5", dev_dependency = True)
 bazel_dep(name = "score_toolchains_rust", version = "0.10.0", dev_dependency = True)
-bazel_dep(name = "score_tools", version = "0.0.3", dev_dependency = True)
+bazel_dep(name = "score_tools", version = "0.0.3")
 
 # Pinned past v0.0.3 for the `$(locations)` fix in score_pytest.
 git_override(
