diff --git a/MODULE.bazel b/MODULE.bazel
index 45cd5c0..da5aa9e 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -66,8 +66,8 @@ bazel_dep(name = "platforms", version = "1.0.0")
 ## S-CORE bazel registry
 bazel_dep(name = "score_baselibs", version = "0.2.4")
 bazel_dep(name = "score_bazel_platforms", version = "0.0.4")
+bazel_dep(name = "score_docs_as_code", version = "3.0.0")
 
-bazel_dep(name = "score_docs_as_code", version = "3.0.0", dev_dependency = True)
 bazel_dep(name = "score_platform", version = "0.5.3", dev_dependency = True)
 bazel_dep(name = "score_process", version = "1.4.3", dev_dependency = True)
 
