diff --git a/third_party/mbedtls/BUILD.bazel b/third_party/mbedtls/BUILD.bazel
index b49a1e1..bcf9fd7 100644
--- a/third_party/mbedtls/BUILD.bazel
+++ b/third_party/mbedtls/BUILD.bazel
@@ -21,7 +21,7 @@ cmake(
     cache_entries = {
         # Force ar instead of libtool to avoid rules_foreign_cc wrapper issues on macOS
         "CMAKE_AR": "/usr/bin/ar",
-        "CMAKE_C_FLAGS": "-DMBEDTLS_SSL_RECORD_SIZE_LIMIT -DMBEDTLS_SSL_NULL_CIPHERSUITES -DMBEDTLS_ED25519_PSA_DRIVER -I$$EXT_BUILD_ROOT$$/comm-mbedtls/mbedtls-sys/csrc",
+        "CMAKE_C_FLAGS": "-DMBEDTLS_SSL_RECORD_SIZE_LIMIT -DMBEDTLS_SSL_NULL_CIPHERSUITES -DMBEDTLS_ED25519_PSA_DRIVER -I$$EXT_BUILD_ROOT$$/external/classic-diagnostic-adapter+/comm-mbedtls/mbedtls-sys/csrc",
         "ENABLE_PROGRAMS": "OFF",
         "ENABLE_TESTING": "OFF",
         "GEN_FILES": "OFF",
