diff --git a/MODULE.bazel b/MODULE.bazel
index ee9ccaf..f899b36 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -365,7 +365,7 @@     commit = "1ce9b563e4a6f6b648ca313967fb2beb10f00317",
     remote = "https://github.com/eclipse-score/itf.git",
 )
 
-bazel_dep(name = "score_rules_imagefs", version = "0.1.0", dev_dependency = True)
+bazel_dep(name = "score_rules_imagefs", version = "0.1.0")
 
 imagefs = use_extension("@score_rules_imagefs//extensions:imagefs.bzl", "imagefs", dev_dependency = True)
 imagefs.toolchain(
