diff --git a/MODULE.bazel b/MODULE.bazel
index ee9ccaf..f899b36 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -430,7 +430,7 @@ 
 # TRLC dependency for requirements traceability
 bazel_dep(name = "trlc", version = "3.0.1", dev_dependency = True)
 bazel_dep(name = "rules_oci", version = "2.2.7", dev_dependency = True)
-bazel_dep(name = "rules_pkg", version = "1.2.0", dev_dependency = True)
+bazel_dep(name = "rules_pkg", version = "1.2.0")
 
 http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 