diff --git a/MODULE.bazel b/MODULE.bazel
index ee9ccaf..f899b36 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -186,7 +186,7 @@     dev_dependency = True,
     lock = "@ubuntu24_04_sysroot_resolve//:lock.json",
 )
 
-bazel_dep(name = "score_cpp_policies", version = "0.1.1", dev_dependency = True)
+bazel_dep(name = "score_cpp_policies", version = "0.1.1")
 bazel_dep(name = "toolchains_llvm", version = "1.8.0", dev_dependency = True)
 
 llvm = use_extension(