diff --git a/score/config_management/config_provider/code/config_provider/details/BUILD b/score/config_management/config_provider/code/config_provider/details/BUILD
index 937b056..9a71294 100644
--- a/score/config_management/config_provider/code/config_provider/details/BUILD
+++ b/score/config_management/config_provider/code/config_provider/details/BUILD
@@ -21,6 +21,7 @@ cc_library(
         "@score_baselibs//score/concurrency:condition_variable",
         "@score_baselibs//score/concurrency/future",
         "@score_baselibs//score/language/futurecpp",
+        "@score_baselibs//score/string_manipulation:string_comparison_adaptor",
         "@score_communication//score/mw/service:proxy_data",
         "@score_logging//score/mw/log",
     ],
diff --git a/score/config_management/config_provider/code/config_provider/details/config_provider_impl.h b/score/config_management/config_provider/code/config_provider/details/config_provider_impl.h
index 7630e60..596a8e6 100644
--- a/score/config_management/config_provider/code/config_provider/details/config_provider_impl.h
+++ b/score/config_management/config_provider/code/config_provider/details/config_provider_impl.h
@@ -19,8 +19,8 @@
 #include "score/config_management/config_provider/code/persistency/persistency.h"
 #include "score/config_management/config_provider/code/proxies/internal_config_provider.h"
 
-#include "score/memory/string_comparison_adaptor.h"
 #include "score/mw/log/logger.h"
+#include "score/string_manipulation/string_comparison_adaptor.h"
 
 #include "score/concurrency/condition_variable.h"
 #include "score/mw/service/proxy_data.h"
