diff --git a/score/launch_manager/src/daemon/src/osal/ipc_comms.hpp b/score/launch_manager/src/daemon/src/osal/ipc_comms.hpp
index 725dc62b..a1b2c3d4 100644
--- a/score/launch_manager/src/daemon/src/osal/ipc_comms.hpp
+++ b/score/launch_manager/src/daemon/src/osal/ipc_comms.hpp
@@ -70,7 +70,7 @@ struct IpcCommsSync final
     /// @brief Constant for the synchronization file descriptor.
     /// The `sync_fd` is a constant representing the file descriptor used for synchronization
     /// during communication. It is set to a value of 3 by default.
-    static const int sync_fd = 3;
+    static const int sync_fd = 111;

     /// @brief Constant for the file descriptor used to signal state transitions
     /// The semaphore used to signal state transitions is stored in an unlinked shared memory area. This requires
