diff --git a/MODULE.bazel b/MODULE.bazel
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -46,2 +46,2 @@ bazel_dep(name = "score_rust_policies", version = "0.0.5", dev_dependency = True)
-bazel_dep(name = "score_process_description", version = "2.1.2", dev_dependency = True)
-bazel_dep(name = "score_platform", version = "0.7.1", dev_dependency = True)
+bazel_dep(name = "score_process_description", version = "2.1.2")
+bazel_dep(name = "score_platform", version = "0.7.1")
