diff --git a/MODULE.bazel b/MODULE.bazel
index a302a84..7917e2b 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -60,7 +60,7 @@ deb = use_repo_rule("@download_utils//download/deb:defs.bzl", "download_deb")
 deb(
     name = "acl-deb",
     build = "//third_party/acl:acl.BUILD",
-    urls = ["https://archive.ubuntu.com/ubuntu/pool/main/a/acl/libacl1-dev_2.2.52-3build1_amd64.deb"],
+    urls = ["https://ports.ubuntu.com/pool/main/a/acl/libacl1-dev_2.2.52-3build1_arm64.deb"],
     visibility = ["//visibility:public"],
 )

