diff --git a/score/datarouter/BUILD b/score/datarouter/BUILD
index 8a986a39..00000000 100644
--- a/score/datarouter/BUILD
+++ b/score/datarouter/BUILD
@@ -899,6 +899,7 @@ cc_binary(
         ":datarouter_app",
         "@score_baselibs//score/os:errno_logging",
         "@score_baselibs//score/os:pthread",
         "@score_baselibs//score/os/utils:path",
         "@score_baselibs//score/mw/log",
+        "//score/mw/log/backend:remote",
         # "//third_party/jemalloc", # Ticket-231781
     ],
 )
