diff --git a/MODULE.bazel b/MODULE.bazel
index bedcacc..7eadc3c 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -47,8 +47,8 @@ single_version_override(
 
 bazel_dep(name = "score_rust_policies", version = "0.0.5")
 
-bazel_dep(name = "score_process_description", version = "2.1.2", dev_dependency = True)
-bazel_dep(name = "score_platform", version = "0.7.1", dev_dependency = True)
+bazel_dep(name = "score_process_description", version = "2.1.2")
+bazel_dep(name = "score_platform", version = "0.7.1")
 bazel_dep(name = "score_itf", version = "0.5.0", dev_dependency = True)
 
 # OCI / Docker image rules for integration tests
