diff --git a/MODULE.bazel b/MODULE.bazel
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -47,8 +47,8 @@ single_version_override(

 bazel_dep(name = "score_rust_policies", version = "0.0.5")

-bazel_dep(name = "score_process", version = "1.5.4", dev_dependency = True)
-bazel_dep(name = "score_platform", version = "0.6.0", dev_dependency = True)
+bazel_dep(name = "score_process", version = "1.5.4")
+bazel_dep(name = "score_platform", version = "0.6.0")
 bazel_dep(name = "score_itf", version = "0.4.0", dev_dependency = True)

 # OCI / Docker image rules for integration tests
