diff --git a/MODULE.bazel b/MODULE.bazel
index 5eed2f9..06fb222 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -47,7 +47,7 @@ bazel_dep(name = "score_itf", version = "0.4.0", dev_dependency = True)
 
 # OCI / Docker image rules for integration tests
 bazel_dep(name = "rules_oci", version = "2.2.7", dev_dependency = True)
-bazel_dep(name = "rules_pkg", version = "1.2.0", dev_dependency = True)
+bazel_dep(name = "rules_pkg", version = "1.2.0")
 
 # QNX image filesystem rules for QNX integration tests
 bazel_dep(name = "score_rules_imagefs", version = "0.0.1", dev_dependency = True)
