diff --git a/MODULE.bazel b/MODULE.bazel
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -102,7 +102,7 @@ bazel_dep(name = "score_toolchains_rust", version = "0.10.0", dev_dependency = T
 bazel_dep(name = "score_docs_as_code", version = "8.0.1")
 bazel_dep(name = "score_process_description", version = "2.1.2")
 
-bazel_dep(name = "score_tooling", version = "2.2.1", dev_dependency = True)
+bazel_dep(name = "score_tooling", version = "2.2.1")
 
 bazel_dep(name = "score_platform", version = "0.7.1")
 
