diff --git a/BUILD b/BUILD
index 492e70606..9c626ac5a 100644
--- a/BUILD
+++ b/BUILD
@@ -11,7 +11,6 @@
 # SPDX-License-Identifier: Apache-2.0
 # *******************************************************************************
 
-load("@hedron_compile_commands//:refresh_compile_commands.bzl", "refresh_compile_commands")
 load("@score_docs_as_code//:docs.bzl", "docs")
 load("@score_tooling//:defs.bzl", "copyright_checker", "dash_license_checker")
 load("@score_tooling//third_party/format:macros.bzl", "use_format_targets")
@@ -98,17 +97,6 @@ docs(
     source_dir = "docs",
 )
 
-# Generate `compile_commands.json`.
-# Required for `clangd` support.
-refresh_compile_commands(
-    name = "generate_compile_commands",
-    exclude_external_sources = True,
-    target_compatible_with = ["@platforms//os:linux"],
-    targets = {
-        "//...": "",
-    },
-)
-
 # Generate `rust_project.json`.
 # Required for `rust-analyzer` support.
 alias(
